Fix the K-Cor Map source to default to MLSO geographic location (#6447)
sunpy/sunpy: changelog/6447.bugfix.1.rst

> Fixed a bug where the private attribute ``_default_observer_coordinate`` for `~sunpy.map.GenericMap` was being used even when there was sufficient observer metadata in the header.

Reviewer: I would maybe call this a `trivial` rather than `bugfix` entry, as it's not public facing? (And you have an entry for the public bug?)

> Fixed a bug where the private property ``_default_observer_coordinate`` for `~sunpy.map.GenericMap` that is used to specify a custom default observer in the absence of sufficient observer metadata was being used even when there was sufficient observer metadata.

Reviewer 2: ```suggestion
Fixed a bug where the private property ``_default_observer_coordinate`` for `~sunpy.map.GenericMap` was being used even when there was sufficient observer metadata in the header.
```
Otherwise that was a very long sentence

Author: Sure
